diff -urN -x CYGWIN-PATCHES -x 'aclocal.m4*' -x autom4te.cache -x config.cache -x config.log -x config.status -x config.h -x config.h.in -x ABOUT-NLS -x Makefile.in.in -x Makevars.template -x '*SlackBuild*' -x '*.class' -x '*.pyc' -x '*.mo' -x '*.gmo' -x '*.orig' -x '*.rej' -x '*.spec' -x '*.temp' -x '*~' -x '*.stackdump' -x COPYING -x INSTALL -x compile -x config-ml.in -x config.guess -x config.sub -x depcomp -x elisp-comp -x install-sh -x ltmain.sh -x mdate-sh -x missing -x mkinstalldirs -x py-compile -x symlink-tree -x texinfo.tex -x ylwrap -x Makefile.in -x makefile.in -x configure -x omf.make -x xmldocs.make -x gnome-doc-utils.make -x gnome-doc-utils.m4 -x intltool.m4 -x intltool-extract -x intltool-extract.in -x intltool-merge -x intltool-merge.in -x intltool-update -x intltool-update.in -x config origsrc/cgdb-0.6.4/Makefile.am src/cgdb-0.6.4/Makefile.am
--- origsrc/cgdb-0.6.4/Makefile.am	2007-04-02 13:00:23.000000000 -0500
+++ src/cgdb-0.6.4/Makefile.am	2007-05-08 07:56:13.140625000 -0500
@@ -4,6 +4,8 @@
 SUBDIRS=lib various tgdb cgdb doc test
 DIST_SUBDIRS=lib various tgdb cgdb doc test
 
+ACLOCAL_AMFLAGS = -I config
+
 # The order is important here.
 # various is the lowest level, so it must be done first.
 # TGDB uses various, so it is done next.
