--- origsrc/cgdb-0.6.4/Makefile.am	2007-04-02 13:00:23.000000000 -0500
+++ src/cgdb-0.6.4/Makefile.am	2007-05-08 07:56:13.140625000 -0500
@@ -4,6 +4,8 @@
 SUBDIRS=lib various tgdb cgdb doc test
 DIST_SUBDIRS=lib various tgdb cgdb doc test
 
+ACLOCAL_AMFLAGS = -I config
+
 # The order is important here.
 # various is the lowest level, so it must be done first.
 # TGDB uses various, so it is done next.
